server/googleAuthStrategies.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+
```

Comment:
Please take care of this extra line

Reply:
fixed in new [commit](https://github.com/opentok/opentok-rtc/pull/675/commits/be36d809175b8545f1fec2276f4b8857cae0dfb9)